deploy/FlapperInit.sol

```diff
@@ -92,6 +87,7 @@ struct SplitterConfig {
 
 library FlapperInit {
     uint256 constant WAD = 10 ** 18;
+    uint256 constant RAY = 10 ** 45;
```

Comment:
```suggestion
    uint256 constant RAD = 10 ** 45;
```

Reply:
This comes from the audit, they should change it there. Also they mention `hump` instead of `bump` in the description.

Reply:
Ah I actually meant RAY here. I think RAY is correct as the no-dust condition is `(tot/RAY - lot + lot) * RAY == tot` which is satisfied if `tot` is a multiple of `RAY`.

Reply:
Ah right, resolving.